resources/issues.md

```diff
@@ -33,3 +33,11 @@ Why? People often ask to be assigned and never come back, which discourages the
 We will accept and merge the first PR that fixes correctly and implements the issue following the contribution guidelines of the corresponding repository.
 
 We are looking forward to your contributions. No need to wait for an assignment to start! 👍
+
+## Issue priority feedback policy based on tier level
+
+Hey \@username.
+No reason why we shouldn't implement this, it's a good idea!
+
+I will add it to the stack of new features to add in this plugin, but we don't have enough time to work on it in the following weeks. 
```

Comment:
Depending on the functionality:
"Unfortunately, the integration team won't have time to implement this improvement. I will create an issue so the community can easily implement it."
OR
"Unfortunately, the integration team won't have time to implement this improvement. I will add some details to this issue so the community can easily implement it."
WDYT?

Reply:
I think the first one does not work because the contributor already created the issue! But the last one I will use to improve the current message :D